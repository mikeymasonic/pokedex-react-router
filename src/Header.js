import React, { Component } from "react";

export default class Header extends Component {
  
  render() {
    return (
      <div>
        <header>
          <a href = "../"><img className="logo" src="https://i.ebayimg.com/images/g/8bMAAOSwxp9W-TVs/s-l400.jpg" alt="Pokemon" /></a>
          <h1>Pokedex</h1>
          </header>
      </div>   
    );
  }
}
