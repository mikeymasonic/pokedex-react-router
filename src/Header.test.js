import React from 'react';
import Header from './Header';
import renderer from "react-test-renderer";
test('renders Header.js correctly', () => {
  const tree = renderer
  .create(<Header />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});