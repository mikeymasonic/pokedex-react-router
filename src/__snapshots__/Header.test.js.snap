// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header.js correctly 1`] = `
<div>
  <header>
    <a
      href="../"
    >
      <img
        alt="Pokemon"
        className="logo"
        src="https://i.ebayimg.com/images/g/8bMAAOSwxp9W-TVs/s-l400.jpg"
      />
    </a>
    <h1>
      Pokedex
    </h1>
  </header>
</div>
`;
